README.md

```diff
@@ -24,6 +24,42 @@ We support **ASP.NET Core** application as well as **.NET Core** application.
 
 * Follow [this example](examples/BasicUsage_clr21_RBAC) for Role-based access control (RBAC) enabled Kubernetes clusters.
 
+### Configuration Details
+
+Customize configurations are supported for `v1.0.2+`. There are several ways to customize the settings. For example:
+
+1. By the code:
+    ```csharp
+    services.AddApplicationInsightsKubernetesEnricher(option=> {
+        option.InitializationTimeout = TimeSpan.FromSeconds(15);
+    });
+    ```
+
+2. By `appsettings.json`:
+    ```jsonc
+    {
+        "Logging": {
+            // ...
+        },
+        // Adding the following section
+        "AppInsightsForKubernetes": {
+            "InitializationTimeout": "3.1:12:15.34"

```

Comment:
Is this 3+ days? Maybe a more practical value, like 15 seconds, would be more illustrative...

Reply:
Shoot :-) You are right, it is a bad example in Readme. I'll update it. 👍